examples/docs_snippets/docs_snippets/guides/data-assets/passing-data-assets/passing-data-io-manager.py

```diff
@@ -25,10 +25,12 @@ def combined_data(people, birds):
     # highlight-end
 
 
-defs = dg.Definitions(
-    assets=[people, birds, combined_data],
-    resources={"io_manager": duckdb_io_manager},
-)
+@dg.definitions
```

Comment:
Should this be in a separate resources.py file? (You can ignore this if that wouldn't make sense in the context of the doc.)

Reply:
I worry it might make things harder to read in the docs. We can throw it out to the group when everyone is back and I can do a followup pass to break things up if people feel it should be.